### PR TITLE
build: clean up references to old `master` branch

### PR DESCRIPTION
**Pull request by devversion**
Cleans up all references to the `master` branch we renamed to
`main` across Angular.

### CODE REVIEW

**Review by atscott — DISMISSED** [review]

**Comment by dgp1130** [comment]
@jelbourn maybe a good candidate for a global approval?

File: aio/content/guide/deployment.md [hunk 1]
```diff
@@ -206,7 +206,7 @@ The list is by no means exhaustive, but should provide you with a good starting
...
...s://pages.github.com)                     | You can't [directly configure](https://github.com/isaacs/github/issues/408) the GitHub Pages server, but you can add a 404 page. Copy `index.html` into `404.html`. It will still be served as the 404 response, but the browser will process that page and load the application properly. It's also a good idea to [serve from `docs` on main](https://help.github.com/articles/configuring-a-publishing-source-for-github-pages/#publishing-your-github-pages-site-from-a-docs-folder-on-your-master-branch) and to [create a `.nojekyll` file](https://www.bennadel.com/blog/3181-including-node-modules-and-vendors-folders-in-your-github-pages-site.htm)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
```
Comment (gkalpak) [inline aio/content/guide/deployment.md:209]: Nit: The `#publishing-your-github-pages-site-from-a-docs-folder-on-your-master-branch` fragment is no longer valid in that GitHub article. Maybe remove or replace with `#choosing-a-publishing-source`:

```diff
-[serve from `docs` on main](https://docs.github.com/en/pages/getting-started-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site/#publishing-your-github-pages-site-from-a-docs-folder-on-your-master-branch)
+[serve from `docs` on main](https://docs.github.com/en/pages/getting-started-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site#choosing-a-publishing-source)
```

File: docs/BRANCHES.md [hunk 2]
```diff
@@ -17,8 +17,8 @@ distribution tags](https://docs.npmjs.com/cli/v6/commands/npm-dist-tag#purpose)
 
 ## Branch naming
 
-Angular's main branch is `master`. This branch always represents the absolute latest changes. The
-code on master always represents a pre-release version, often published with the `next` tag on npm.
+Angular's main branch is `main`. This branch always represents the absolute latest changes. The
+code on main always represents a pre-release version, often published with the `next` tag on npm.
```
Comment (gkalpak) [inline docs/BRANCHES.md:21]: Nit (although not directly related to this PR - feel free to ignore):

```suggestion
code on `main` always represents a pre-release version, often published with the `next` tag on npm.
```

File: docs/BRANCHES.md [hunk 3]
```diff
@@ -92,17 +92,18 @@ The vast majority of pull requests will target `major`, `minor`, or `patch` base
 the code change. In rare cases, a pull request will specify `target: rc` or `target: lts` to
 explicitly target a special branch.
 
-Breaking changes, marked with `target: major`, can only be merged when `master` represents the next
+Breaking changes, marked with `target: major`, can only be merged when `main` represents the next
 major version.
 
+### Pull request examples
```
Comment (gkalpak) [inline docs/BRANCHES.md:98]:
```suggestion
```

File: scripts/compare-main-to-patch.js [hunk 4]
```diff
@@ -152,32 +152,32 @@ function main() {
...
   const patchCommitsMap = collectCommitsAsMap(patchCommits);
 
   // tslint:disable-next-line:no-console
   console.log(`
-Comparing branches "${branch}" and master.
+Comparing branches "${branch}" and main.
 
-***** Only in MASTER *****
-${diff(masterCommitsMap, patchCommitsMap).join('\n') || 'No extra commits'}
+***** Only in MAIN *****
```
Comment (gkalpak) [inline scripts/compare-main-to-patch.js:176]: So, here you _do_ uppercase it 🤔 😛

**Review by gkalpak — DISMISSED** [review]
:tada:

Reviewed-for: global-dev-infra-approvers

File: aio/content/guide/deployment.md (hunk 1 above)
Comment (devversion) [reply aio/content/guide/deployment.md:209]: Good idea, let's incorporate this here as well.

File: scripts/compare-main-to-patch.js (hunk 4 above)
Comment (devversion) [reply scripts/compare-main-to-patch.js:176]: haha! look at my latest fixup, I'm sure you'll like it 😛

Comment (gkalpak) [reply scripts/compare-main-to-patch.js:176]: Love it 🤤 ❤️ 😄

**Review by gkalpak — APPROVED** [review]
Reviewed-for: global-dev-infra-approvers

**Comment by AndrewKushnir** [comment]
This PR was merged into the repository by commit 68a6a075f4aeab1788a1bba736bfcb428cfbbacb.

**Comment by AndrewKushnir** [comment]
@devversion FYI there was a conflict while merging this PR to the patch (13.3.x) branch, so I've merged it into the main and 14.0.x branches. Could you please create a new PR targeting 13.3.x branch? Thank you.

**Pull request closed without merging**